wiki/Bazel-Setup-Instructions-for-Windows.md

```diff
@@ -53,18 +53,21 @@ After that, follow each of the subsections below as needed to install prerequisi
 
 **Java**
 
-JDK 8 is required for the Android build tools, and we suggest installing OpenJDK:
+- JDK >= 17 is required for Android Package Manager.
```

Comment:
This is a good catch. It's unfortunate two versions of Java is needed, and it may cause confusion if people ever need to run sdkmanager after installing JDK8.

For better reference capability, perhaps we should instead:
- Suggest installing JDK 8 & 17 at the same time.
- Always suggest using update-alternatives to default to JDK17 prior to using sdkmanager.
- Always suggest using update-alternatives to default to JDK8 prior to building.

Reply:
Modified the instructions to handle the installations of both JDK at the same time and set default configurations prior to running commands via this commit - https://github.com/oppia/oppia-android/pull/5325/commits/cb63c0701291230bb43610c8f8a00353cf965426